src/Controller/JsonController.php

```diff
@@ -49,15 +52,22 @@ public function checkIn(Guest $guest, Request $request): JsonResponse
     {
         $payload = [];
         if (($content = $request->getContent()) !== '') {
-            $payload = json_decode($content, true, 512, JSON_THROW_ON_ERROR);
+            try {
+                $payload = json_decode($content, true, 512, JSON_THROW_ON_ERROR);
+            } catch (\JsonException $e) {
```

Comment:
I'd return a response here to prevent working with broken data, e.g.
```php
return $this->json([
    'exception' => get_class($e),
    'message' => $e->getMessage(),
    'stack' => $e->getTrace(),
], Response::HTTP_BAD_REQUEST);
```